.circleci/config.yml

```diff
@@ -14,6 +14,8 @@ jobs:
       - run: sudo pip install ansible
       - run: cp ./sonar-scanner.properties sonar-scanner-3.0.1.733/conf/sonar-scanner.properties
       - run: mvn clean verify
+      - run: curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+      - run: sudo apt-get install -y nodejs
```

Comment:
I'm working on another change set that'll introduce a custom build image so as to remove a lot of the dependency installation from the build job configurations.